charts/wadm/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for wadm.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/wasmcloud/wadm
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 1000
+
+securityContext:
+  runAsUser: 1000
+  runAsGroup: 1000
+  runAsNonRoot: true
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - "ALL"
+  seccompProfile:
+    type: "RuntimeDefault"
+
+
+env:
+  - name: WADM_NATS_SERVER
+    value: nats-cluster.default.svc.cluster.local
+  - name: WADM_JETSTREAM_DOMAIN
```

Comment:
I wouldn't set anything as a default here. By default it doesn't use a domain in most setups and that is fine